test-cases/longevity/longevity-mv-si-4days.yaml

```diff
@@ -15,6 +15,7 @@ n_loaders: 2
 n_monitor_nodes: 1
 
 instance_type_db: 'i3.4xlarge'
+gce_instance_type_db: 'n1-highmem-32'
```

Comment:
same here

Reply:
left 32

Reply:
above you use n1-highmem-16 for i3.4xlage, why here n1-32?

Reply:
Because @roydahan asked in the task to add tests with many cores 24/32, so I chose "randomly" some tests where the instance is with 32 cores.